src/clj/game/cards/agendas.clj

```diff
@@ -1174,7 +1177,8 @@
 
    "SSL Endorsement"
    (let [add-credits (effect (add-counter card :credit 9))
-         remove-credits {:optional {:req (req (pos? (get-counters card :credit)))
+         remove-credits {:interactive (req true)
+                         :optional {:req (req (pos? (get-counters card :credit)))
```

Comment:
Same comment as for Corp Sales Team, except the manual ability probably should just take 3 credits instead of triggering an Optional.

Reply:
Done did